flags.go

```diff
@@ -134,9 +134,10 @@ func newApp() (app *cli.App) {
 			/////////////////////////
 
 			cli.StringFlag{
-				Name:  "endpoint",
-				Value: "https://storage.googleapis.com:443",
-				Usage: "The endpoint to connect to.",
+				Name: "custom-endpoint",
+				Usage: "The end point to connect to for fetching data. The endpoint should be equivalent " +
```

Comment:
Sorry for multiple iterations. can you rephrase like this:

Alternate endpoint for fetching data. Should be used only for testing purposes. The endpoint should be equivalent to the base endpoint of GCS JSON API (https://storage.googleapis.com/storage/v1). If not specified GCS endpoint will be used. Auth will be skipped for custom endpoint.